src/app/[locale]/class-based/[id]/assignments/[assignmentId]/details/components/SubmissionCard.tsx

```
@@ -0,0 +1,154 @@
+        <Card>
+            <CardHeader className="flex flex-row items-center justify-between pb-2">
+                <CardTitle className="text-lg font-medium">Bài tập của bạn</CardTitle>
+                <span className="text-sm text-muted-foreground">
+                    {assignmentSubmissionUtils.getStatusLabel(submission.status)}
+                </span>
+            </CardHeader>
+            <CardContent className="space-y-4">
+                <div className="space-y-2">
+                    {files.map((file) => (
+                        <FileItem file={file} />
+                    ))}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)